libraries/standard/json/src/json.c

```diff
@@ -404,7 +422,11 @@ static bool_ skipEscape( const char * buf,
         }
     }
 
-    *start = i;
+    if( ret == true )
```

Comment:
redundant.

Reply:
No, the value of i is changed and must be given back to the caller.